src/native/corehost/test/typelibs/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+
+# Get the current list of definitions to pass to midl
```

Comment:
I am thrilled I didn't have to figure out how to make CMake compile an IDL. I am sad it is something I have now seen and am now aware is possible...

Reply:
You sure you haven't just selectively erased the `coreclr/inc` directory from your memory?

Reply:
Yeah it's not fun to hook this up. Not terrible, but not fun.